src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
@conan-z I checked rebuild-plugin then found it has various jelly files for ParameterValue. It includes one for non-core class. e.g. MavenMetadataParameterValue.jelly for jenkins-maven-metadata-plugin, NodeParameterValue.jelly for nodelabelparameter-plugin.

And RebuildAction in rebuild plugin has a logic to load internal jelly files with the simple name of class. So I think that this issue can be solved without any patches for this plugin.

Maybe, rebuild plugin has changed the policy to support ParameterValue extended by other plugins?

BTW, I cannot find plugins that RebuildParameterProvider has been implemented.(googling...)
Do you know such plugin?